platform/ddf-theme/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I like what this PR is doing, but I don't think we need to add a new maven module. I think we can move a lot of this work under the `ui/` modules so we don't end with the same yarn contention problems we had before by having too many maven modules launching yarn processes.